### ISSUE TITLE
Missing app icon under GNOME

### ISSUE THREAD

**Issue by SteveLauC** [flathub/org.kde.massif-visualizer#41]
Hi, thanks for packaging this tool to Flathub!

I just installed it and found this app does not have an icon, then I tried to install the app packaged by my distribution to see if it is the same:

> I am on Debian 12

```sh
$ sudo apt install massif-visualizer
```

then the icon shows up.

It seems that some packages are missing?

**Comment by SteveLauC** [comment]
Yeah, closing as it should be fixed per the above comment. I cannot test it tough as I no longer daily drive Linux

**Comment by yakushabb** [comment]
<img width="1130" height="836" alt="Image" src="https://github.com/user-attachments/assets/189fe64e-4bd6-4a61-94a7-e2ecf0c0b22f" />
